website/common/script/content/constants/events.js

```diff
@@ -115,16 +124,16 @@ export const EVENTS = {
     gear: true,
   },
   bundle202306: {
-    start:'2023-06-13T08:00-04:00',
-    end:'2023-06-30T23:59-04:00',
+    start: '2023-06-13T08:00-04:00',
+    end: '2023-06-30T23:59-04:00',
```

Comment:
more random autolinter changes. Again, not necessary to remove these rn, but watch out for this in future. This is the sort of thing you should be able to spot instantly by looking at the "changed files" page on your PR.

Reply:
I did not fix these because if you look at the events from 2022 and earlier, there is a space there and I didn't want to send in a bunch of new changes.